features/step_definitions/basic_steps.rb

```diff
@@ -31,6 +31,18 @@ def page_path_from(page_name)
   end
 end
 
-Given("I provide invalid credentials") do
+Given('I provide invalid credentials') do
   OmniAuth.config.mock_auth[:spotify] = :invalid_credentials
 end
+
+Then('I should be on {string} page') do |string|
+  visit concert_tickets_path
```

Comment:
Why dont you use the variable in this step def?
You dont want to hard code this.